Tasks/KubeloginInstallerV0/utils.ts

```diff
@@ -84,6 +90,12 @@ export async function getKubeloginRelease(version: string = 'latest', platform?:
     request.method = 'GET';
     request.headers = request.headers || {};
     request.headers['User-Agent'] = userAgent;
+    if (taskLib.getBoolFeatureFlag('USE_AUTHORIZATION_FOR_API_CALL')){
+    const token = getGithubEndPointToken();
+    if (token) {
+        request.headers['Authorization'] = 'token ' + token;
```

Comment:
Let's extract explicit utility method to reduce code duplication

Reply:
Implemented the utlity code